csharp/rust/src/ffi.rs

```diff
@@ -71,6 +71,8 @@ pub struct ConnectionConfig {
     pub protocol: redis::ProtocolVersion,
     /// zero pointer is valid, means no client name is given (`None`)
     pub client_name: *const c_char,
+    pub has_lazy_connect: bool,
+    pub lazy_connect: bool,
```

Comment:
Remove and/or add TODO, otherwise no one will spot that that feature isn't implemented in C# client.